contrib/epee/src/readline_buffer.cpp

```diff
@@ -167,30 +165,42 @@ static int handle_enter(int x, int y)
   char* line = NULL;
```

Comment:
@jtgrassie I was referring to the assignment copy of the std::string a few lines below this - that can potentially throw `std::bad_alloc`.